shells/pipes-shell-2/web/deploy/source/index.html

```diff
@@ -11,26 +11,46 @@
 <html lang="en">
 <head>
 
-  <title>Pipes (Arcs)</title>
+  <title>Arcs</title>
 
   <meta charset="utf-8">
   <meta name="viewport" content="width=device-width, user-scalable=no, minimum-scale=1.0, maximum-scale=1.0">
 
-  <script type="module" src="./shell.js"></script>
-
   <style>
-    body, input, button {
-      font-family: sans-serif;
+    body {
```

Comment:
Move to a separate PR.

Reply:
Maybe I'm missing something Walt, but this seems ok to me. IMO the point of the `index.html` is high-level configuration, and it seems like all of these changes (styling too) are aimed at tailoring the app for the Android platform, which strikes me as proper.